test/enigma_test.rb

```diff
@@ -113,6 +113,26 @@ def test_it_can_write_to_txt
     File.delete("./text_files/test_message_write.txt")
   end
 
+  def test_it_can_encrypt_from_file
+    filepaths = ["./text_files/test_message.txt", "./text_files/encrypted.txt"]
+    @enigma.stubs(:user_input).returns(filepaths)
+    Time.stubs(:now).returns(Time.new(1995, 04, 8))
+    @enigma.stubs(:generate_number).returns(2715)
+    assert_equal false, File.exists?("./text_files/encrypted.txt")
+    expected = "Created './text_files/encrypted.txt' with the key 02715 and date 040895"
+    assert_equal expected, @enigma.encrypt_from_file
+    assert_equal "keder ohulw!", @enigma.read_from_txt("./text_files/encrypted.txt")
+    File.delete("./text_files/encrypted.txt")
+  end
 
-
+  def test_it_can_decrypt_from_file
+    input = ["./text_files/encrypted_test.txt", "./text_files/decrypted.txt","02715", "040895"]
+    @enigma.stubs(:user_input).returns(input)
+    Time.stubs(:now).returns(Time.new(1995, 04, 8))
+     assert_equal false, File.exists?("./text_files/decrypted.txt")
+    expected = "Created './text_files/decrypted.txt' with the key 02715 and date 040895"
+    assert_equal expected, @enigma.decrypt_from_file
+    assert_equal "hello world!", @enigma.read_from_txt("./text_files/decrypted.txt")
```

Comment:
this line confirms that the contents of the file are the expected message and that the message was decrypted properly.